### DISCUSSION TITLE
Termoregolazione radiante con shelly e HA

### DISCUSSION THREAD

**Discussion by antony84** [indomus/forum#2196] [Hardware & Firmware]
Ciao a tutti,
vorrei provare con il vostro aiuto a capire se posso realizzare un impianto di termoregolazione fai da te per un radiante caldo/freddo multizona.
Le esigenze per la stagione invernale sono:
- in inverno controllo e setpoint temperatura di ogni zona, con attivazione pompa di calore e attivazione/disattivazione testine della zona
- in estate, oltre agli stessi controlli previsti in inverno, sarebbe necessario impostare setpoint UR in %, e dare eventualmente consenso al deumidificatore. In più, in caso di apertura porta/finestra, chiudere testine e togliere consensi a deu e pdc.

Per cui lato hardware l'idea è la seguente. Come sensore T/U per ogni ambiente usare uno shelly plus 1 con lo shelly plus addon, con sonda dht22 collegata. Lo shelly plus addon ha inoltre un ingresso digitale per poter segnalare con un magnete l'eventuale apertura della porta/finestra.
In base a determinati livelli di T/H andrei a comandare le testine con lo stessto contatto pulito dello shelly plus 1, e a comandare pompa di calore e deumidificatore con altrettanti shelly 1 collegati a questi due macchinari e azionati tramite contatti puliti.

Che ne pensate? La ritenete una cosa fattibile?

Se lato hw è fattibile, lato software potrei effettuare queste regolazioni articolate con con home assistant? E ad esempio prevedere due profili estate/inverno?

Grazie

**Comment by antonellopasella** [comment]
Impianto radiante ha isteresi molto alta, ergo é inutile abbassare la temperatura alle 21, perché vedresti il cambiamento alle 4 di mattina e poi alle 6 hai freddo.

Il radiante non si regola,verifica col progettista del tuo impianto e ti dirà lo stessi.


Un’applicazione reale (ma complicata) é quella di basarsi sulle previsioni meteo per alzare in anticipo.

esempio: se domani la temperatura dovrebbe scendere di 7 gradi, magari si può tenere più alto per qualche ora per non farsi cogliere imprepaeati.

ma, ripeto, non é una cosa che risplvi con una automazione e un template
